awx/ui/src/components/PersistentFilters/PersistentFilters.js

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { useLocation, useHistory } from 'react-router';
+
+const STORAGE_KEY = 'persistentFilter';
```

Comment:
We could this one to the constant file. Also, the other constants used on session storage starts with awx to make ease to identify them.